docs/examples/rx/02_listen_to_blocks/index.js

```diff
@@ -0,0 +1,15 @@
+// Import the API
+const { ApiRx } = require('@polkadot/api');
+
+async function main () {
+  // Here we don't pass the (optional) provider, connecting directly to the default
+  // node/port, i.e. `ws://127.0.0.1:9944`. Await for the isReady promise to ensure
+  // the API has connected to the node and completed the initialisation process
+  ApiRx.create().subscribe((api) => {
```

Comment:
Ah, ok. I copied the subscribe inside subscribe pattern from the ApiRx chapter, we definitely need to change it there, too. https://github.com/polkadot-js/api/blob/master/docs/api/classes/_rx_index_.apirx.md#constructor

So it would be this?
```
const api = new ApiRx();
api.rpc.chain.subscribeNewHead().subscribe((header) => {
  console.log(`Chain is at block: #${header.blockNumber}`);
});
```

What happened to .isReady btw, that"S not needed anymore?

Reply:
Ah... I overread the switchmap-part...

Reply:
Changed, still needs to be fixed in the classes example, though